kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeAppCell.js

```diff
@@ -12,54 +12,57 @@
  */
 
 (function( $, undefined ) {
-  require(['kbwidget', 'bootstrap', 'jquery',
-           'narrativeConfig',
-           'util/string',
-           'util/display',
-           'util/bootstrapDialog',
-           'kbaseAuthenticatedWidget',
-           'kbaseAccordion',
-           'kbaseNarrativeOutputCell'
-           ],
-  function(KBWidget, bootstrap,$,
-           Config,
-           StringUtil,
-           DisplayUtil,
-           BootstrapDialog,
-            kbaseAuthenticatedWidget, kbaseAccordion, kbaseNarrativeOutputCell) {
     'use strict';
-    return KBWidget({
-        name: "kbaseNarrativeAppCell",
-        parent : kbaseAuthenticatedWidget,
-        version: "1.0.0",
-        options: {
-            app: null,
-            cellId: null,
-            loadingImage: Config.get('loading_gif'),
-            methodStoreURL: Config.url('narrative_method_store'),
-
-            appHelpLink: '/#appcatalog/app/l.a/',
-            methodHelpLink: '/#appcatalog/app/l.m/' // apps cannot have SDK methods, so always add the legacy module id
-        },
-        IGNORE_VERSION: true,
-        defaultInputWidget: 'kbaseNarrativeMethodInput',
-        defaultOutputWidget: 'kbaseDefaultNarrativeOutput',
-
-        appSpec: null,
-        methodSpecs: null,
-        inputSteps: null,
-        inputStepLookup: null,
-
-        $runButton: null,
-        $stopButton: null,
-
-        errorDialog: null,
-
-        OUTPUT_ERROR_WIDGET: 'kbaseNarrativeError',
-
-        state: null,
-
-        /**
+    require(['kbwidget',
```

Comment:
A lot of whitespace changes in this file. Do you know why it is written as an IIFE, unlike most of the other narrative code we've come across so far?

Next comment will be at my code changes (as opposed to whitespace reformatting).